.github/workflows/build.yml

```diff
@@ -5,9 +5,14 @@ on: [push]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        postgres_version: ['12', '13', '14']
+        python-version: [3.10.6]
+        redis-version: [6]
```

Comment:
where is this used?

Reply:
deleting this PR, it was based off an outdated branch